ci.sh

```diff
@@ -2,7 +2,7 @@
 set -ex
 
 # Fixed specr-transpile version
-VERSION="0.1.16"
+VERSION="0.1.18"
```

Comment:
This required some more changes, please just rebase over master. :)

Reply:
Done. But we get a new warning, will look into it.
```
warning: ambiguous glob re-exports
  --> /minirust/tooling/miniutil/src/lib.rs:9:9
   |
9  | pub use gen_minirust::prelude::*;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^ the name `NdResult` in the type namespace is first re-exported here
...
13 | pub use gen_minirust::libspecr::*;
   |         ------------------------- but the name `NdResult` in the type namespace is also re-exported here
   |
```

Reply:
Yeah we have an overlap between the libspecr prelude and the minirust prelude. Not sure what the best solution is. We do want the minirust one to take precedence though so we should probably at least reorder the imports.

AFAIK explicit imports take precedence, so maybe we can just add `pub use gen_minirust::prelude::NdResult` before the `prelude::*` import?